jsoncpp/PSPBUILD

```diff
@@ -9,6 +9,14 @@ makedepends=()
 source=("git+https://github.com/open-source-parsers/jsoncpp#commit=5defb4ed1a4293b8e2bf641e16b156fb9de498cc")
 sha256sums=('SKIP')
 
+prepare() {
+    cd "$pkgname/pkg-config"
+    sed -i 's#@CMAKE_INSTALL_PREFIX@#${PSPDEV}/psp#' jsoncpp.pc.in
```

Comment:
```suggestion
    sed -i 's#@CMAKE_INSTALL_PREFIX@#${PSPDEV}/psp#g' jsoncpp.pc.in
```
I'd add this just to make sure it applies to both instances, even though it already seems to.